app/__init__.py

```diff
@@ -14,30 +18,50 @@
 search_client = SearchClient.create(Config.ALGOLIA_APP_ID, Config.ALGOLIA_API_KEY)
 index = search_client.init_index(Config.INDEX_NAME)
 
+app = Flask(__name__, static_folder='app/static')
+app.config.from_object(Config)
+app.url_map.strict_slashes = False
 
-def create_app(config_class=Config):
-    app = Flask(__name__, static_folder=None)
-    app.config.from_object(config_class)
-    app.url_map.strict_slashes = False
+limiter = Limiter(
+    app,
+    key_func=get_remote_address,
+    default_limits=["200 per day", "50 per hour"]
+)
 
-    Limiter(
-        app,
-        key_func=get_remote_address,
-        default_limits=["200 per day", "50 per hour"]
-    )
+db.init_app(app)
+migrate.init_app(app, db)
 
-    db.init_app(app)
-    migrate.init_app(app, db)
+from app.api import bp as api_bp # noqa
+app.register_blueprint(api_bp, url_prefix='/api/v1')
 
-    from app.api import bp as api_bp
-    app.register_blueprint(api_bp, url_prefix='/api/v1')
+from app.views import bp as view_bp # noqa
+app.register_blueprint(view_bp)
 
-    from app.views import bp as view_bp
-    app.register_blueprint(view_bp)
+from app.errors import bp as error_bp # noqa
+app.register_blueprint(error_bp)
 
-    from app.errors import bp as error_bp
-    app.register_blueprint(error_bp)
 
-    add_health_check(app)
+@app.route("/healthz")
+@limiter.exempt
```

Comment:
Can we add a unit test for the `/healthz` endpoint that makes many requests and verifies that it's not rate limited anymore? I saw that it works I just want to be able to verify that future changes don't break this again

Reply:
I'm not sure how I would do that in a unit test. do you have any ideas on how to implement that?

Reply:
There's a rate limiting unit test already. You can follow that same pattern but change the assertion that it's still returning the expected response rather than the 429

https://github.com/OperationCode/resources_api/blob/master/tests/unit/test_routes/test_utils.py#L107

Reply:
Let me know if you need additional help with this and I can write the test. Not trying to cop out and make you do it all, just have been really busy this week. I'll have some time this weekend

Reply:
test added